Extensions/xkit_updates.js

```diff
@@ -292,7 +292,7 @@ XKit.extensions.xkit_updates = new Object({
 				try {
 					var mdata = JSON.parse(response.responseText);
 
-					if (mdata.malicious === true || mdata.malicious == "true") {
+					if (mdata.malicious || mdata.malicious == "true") {
```

Comment:
Fairly sure nothing, in theory if (x === true) and if (x) should mean the same thing, namely checking a boolean non-string value for trueness.